src/components/BottomSheet.tsx

```diff
@@ -38,6 +39,10 @@ function BottomSheet({
   const [pickerHeight, setPickerHeight] = useState(0)
   const safeAreaInsets = useSafeAreaInsets()
 
+  useEffect(() => {
+    if (isVisible) Keyboard.dismiss()
```

Comment:
nit: I'm a bit concern about this being triggered when we don't want to...
What would happen if there is a component with `autoFocus` in the BottomSheet? So we will expect to have the keyboard ready.
Maybe we should leave this logic to the BottomSheet caller?
Anyways, I think this is a fine.

Reply:
We should not be putting components with text inputs within bottom sheets in general. I don't see in a place in the code where we are calling `autoFocus={true}` on a component in a bottom sheet currently so I think this is fine for now too.